packages/react/src/combobox/input/ComboboxInput.tsx

```diff
@@ -163,62 +163,7 @@ export const ComboboxInput = React.forwardRef(function ComboboxInput(
             fieldControlValidation?.commitValidation(valueToValidate);
           }
         },
-        onCompositionStart(event) {
-          isComposingRef.current = true;
-          setComposingValue(event.currentTarget.value);
-        },
-        onCompositionEnd(event) {
-          isComposingRef.current = false;
-          const next = event.currentTarget.value;
-          setComposingValue(null);
-          store.state.setInputValue(
-            next,
-            createBaseUIEventDetails('input-change', event.nativeEvent),
-          );
-        },
```

Comment:
cc: @mj12albert to test with composed characters, none of this should be necessary now